lux/utils/sorters.py

```diff
@@ -0,0 +1,31 @@
+#  Copyright 2019-2020 The Lux Authors.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+class Sorter:
+    @staticmethod
+    def interestingness():
```

Comment:
Wondering if we should just merge and rename the Interestingness.py class to a Sorter. With the sorter, we should have the option to pick different interestingness metrics, such as correlation, deviation, etc (while not all of them may be applicable for different actions).